CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/geospatial/AbstractGeoExportPlugin.java

```diff
@@ -461,8 +461,11 @@ public void read(final GraphReadMethods graph, final PluginInteraction interacti
                 throw new PluginException(PluginNotificationLevel.ERROR, "Invalid element type");
         }
 
-        try {
-            exportGeo(parameters, GraphNode.getGraphNode(graph.getId()).getDisplayName(), shapes, attributes, output);
+        try {            
+            //#1646: Check for valid path
+            if(isValidPath(output)) {

```

Comment:
are you able to add a space between `if` and `(`?

Reply:
Done